handsontable/src/plugins/pagination/pagination.js

```diff
@@ -260,10 +277,79 @@ export class Pagination extends BasePlugin {
 
     this.#ui.destroy();
     this.#ui = null;
+    this.unregisterShortcuts();
 
     super.disablePlugin();
   }
 
+  /**
+   * Register shortcuts responsible for navigating through the pagination.
+   *
+   * @private
+   */
+  registerShortcuts() {
+    const manager = this.hot.getShortcutManager();
+    const pluginContext = manager.getContext(SHORTCUTS_CONTEXT_NAME) ??
+      manager.addContext(SHORTCUTS_CONTEXT_NAME, 'global');
+
+    pluginContext.addShortcut({
+      keys: [['Shift', 'Tab'], ['Tab']],
+      preventDefault: false,
+      callback: (event) => {
+        const { activeElement } = this.hot.rootDocument;
+        const index = this.#ui.focusableElements.indexOf(activeElement);
+
+        if (event.shiftKey) {
+          const reversedIndex = this.#ui.focusableElements.length - 1 - index;
```

Comment:
As in the previous comment, it is not the best idea to have a `ui.js` module that separates DOM manipulation, and here that manipulation leaks to the `pagination.js` file, which should hold only initialization stuff. Can I ask you to move it to a separate file?